File: src/list/findMineLocation.ts
export type Minefield = Array<number[]>;

export default function findMineLocation(minefield: Minefield): [row: number, column: number] {
  for (let index = 0; index < minefield.length; index++) {
    const column = minefield[index].indexOf(1);
    if (column > -1) return [index, column];
  }
  throw new Error("Field does not contain a bomb! (Lucky!)");
}

File: src/list/find_mine_location.py
def find_mine_location(minefield: list[tuple[int, int]]) -> tuple[int, int]:
  for index, row in enumerate(minefield):
    for idx, tile in enumerate(row):
      if tile == 1:
        return [index, idx]
  raise ValueError("Field does not container a bomb! (Lucky!)")